### ISSUE TITLE
✅ 🔗 🐞 Bug Report: Conditionals for Popups Missing Beta New UI (Related to #22050)

### ISSUE THREAD

**Issue by maurocarreraok** [elementor/elementor#21675]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The conditionals button is missing on the Popup Builder using the Revamped UI
<img width="3008" alt="Screenshot 2023-03-16 at 16 04 25" src="https://user-images.githubusercontent.com/68023943/225727563-84891f12-4f26-43e8-a9e4-a8dd02d978b9.png">

### Steps to reproduce

Create a popup
No conditionals button to edit after the first prompt.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.39-42
	PHP Version: 8.1.16
	PHP Memory Limit: 768M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://test.c42.studio
	Home URL: https://test.c42.studio
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: es
	Timezone: 0
	Admin Email: wp@c42.studio
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.0
	Author: Crocoblock
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: es_ES
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.13.0-dev2
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.0-beta2
		Author: Elementor.com

	UiPress Lite
		Version: 3.1.0
		Author: Admin 2020

	UiPress Pro
		Version: 3.0.7
		Author: Admin 2020


== Uso de los elementos ==
	
	header : 1
		container : 5
		mega-menu : 1
		theme-site-logo : 1
		video : 1
	popup : 1
		button : 1
		container : 2
		heading : 2
		image : 1
		text-editor : 1


== Ajustes ==
	
	cpt_support: post, page
	allow_tracking: yes
	font_display: swap


== Caracteristicas ==
	Custom Fonts: 0
	Custom Icons: 0

== Integraciones ==
	


== Experimentos de Elementor ==
	Salida del DOM optimizada: Activo
	Carga de recursos mejorada: Activo
	Carga de CSS mejorada: Activo
	Fuentes de iconos integradas: Activo
	Mejoras de accesibilidad: Activo
	Puntos de ruptura personalizados adicionales: Activo
	admin_menu_rearrangement: Inactivo por defecto
	Contenedor Flexbox: Activo
	Actualizar la librería Swiper: Activo
	Container Grid: Inactivo por defecto
	Por defecto, el nuevo constructor de temas: Activo
	Cabecera y pie de página del tema Hello: Activo
	Editor Top Bar: Activo
	Páginas de destino: Activo
	Elementos anidados: Activo
	Carga diferida de imágenes de fondo: Activo
	Transiciones de páginas: Activo
	Notas: Activo
	Bucle: Activo
	Ajuste de scroll: Activo
	Menú: Activo


== Registro ==
	
Log: showing 20 of 362023-03-06 14:49:53 [info] elementor::elementor_updater Started 
2023-03-06 14:49:53 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-06 14:49:53 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-06 14:49:53 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.2',
  'to' => '3.13.0-dev1',
)]
2023-03-06 14:49:53 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.2',
  'to' => '3.13.0-dev1',
)]
2023-03-06 14:51:21 [info] elementor-pro::elementor_pro_updater Started 
2023-03-06 14:51:21 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-06 14:51:21 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-06 14:51:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.3',
  'to' => '3.12.0-beta1',
)]
2023-03-06 14:51:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.3',
  'to' => '3.12.0-beta1',
)]
2023-03-16 18:58:34 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.0-dev1',
  'to' => '3.13.0-dev2',
)]
2023-03-16 18:58:35 [info] elementor::elementor_updater Started 
2023-03-16 18:58:35 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-16 18:58:35 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-16 18:58:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.0-dev1',
  'to' => '3.13.0-dev2',
)]
2023-03-16 19:03:48 [info] elementor-pro::elementor_pro_updater Started 
2023-03-16 19:03:48 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-16 19:03:48 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-16 19:03:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.0-beta1',
  'to' => '3.12.0-beta2',
)]
2023-03-16 19:03:48 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.0-beta1',
  'to' => '3.12.0-beta2',
)]

PHP: showing 5 of 5PHP: 2023-03-06 14:47:49 [notice X 18][/home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/plugin.php(154): class type include()array (
  0 => '/home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/plugin.php(154): class type include()array (
  0 => '/home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(243): ElementorPro\Plugin -> autoload()
',
)]
PHP: 2023-03-06 14:48:01 [warning X 3][/home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-06 14:49:04 [notice X 237][/home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-06 14:54:33 [warning X 4][/home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-06 14:55:57 [error X 1][/home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/customer/www/test.c42.studio/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /home/customer/www/test.c42.studio/public_html/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /home/customer/www/test.c42.studio/public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /home/customer/www/test.c42.studio/public_html/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /home/customer/www/test.c42.studio/public_html/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2023-03-06 14:54:19 [error X 2][https://test.c42.studio/wp-content/plugins/elementor/assets/js/df16c9d56464dd4f0800.bundle.min.js:2:7826] Cannot read properties of null (reading 'dataset') 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibilidad no especificada

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by nicholaszein** [comment]
Hi there @maurocarreraok, nice to see you here!

The new Top Bar is still an alpha preview, and not all features are available yet, including Display Conditions. But we have news!

We are happy to announce that this new feature is now live in the latest v3.13-Beta release!

We would love to see you try this new feature and give your feedback in the [Beta Announcement](https://github.com/orgs/elementor/discussions/22119)! 💪

Please try it out and let us know!

https://github.com/orgs/elementor/discussions/22119

Kind regards

PS: If you don't have time to test this out, please consider disabling the Top Bar feature until we release version 3.13.

**Comment by nicholaszein** [comment]
Hey, Mauro @maurocarreraok!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.13.0**! 🥳 

You can now find the options for the Pop-up settings in the new Top Bar.

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
